docs/vsphere_ci/README.md

```diff
@@ -71,7 +71,6 @@ adjust the following variables:
 - `<vsphere-cluster>` Name of the vSphere cluster
```

Comment:
Commit message comment:

> name property added to build.json script for best practices,
   otherwise it default to type and limit the number of builder
   definitions due name collisions

Not sure what this implies and might need rewording.

Reply:
Good catch, actually no need to introduce the `name` property, so comment no longer relevant.
For further references see [packer documentation](https://www.packer.io/docs/templates/legacy_json_templates/builders#named-builds)